### PR TITLE
unify logging content style

### PR DESCRIPTION
**Pull request by sanxun0325**
### Describe what this PR does / why we need it
unify logging content style

### Does this pull request fix one issue?
Fixes:https://github.com/alibaba/sentinel-golang/issues/280

### Describe how you did it
对项目中的四种日志类型统一风格，统一的标准如下：

**Error：**

1. error 描述小写字母开头
2. message 大写字母开头
3. message 结尾声明错误在哪一个func()
4. 输出参数使用keysAndValues
例如:logging.Error(errors.New("bucket data type error"), "Bucket data type error in errorCounterLeapArray.allCounter()","param","paramValue")

**Info:**

1. 日志以[$module]开头
2. message 大写字母开头
3. 输出参数使用keysAndValues
例如:logging.Info("[Config] Print effective global config", "globalConfig", *globalCfg)

**Warn:**

1. 日志以[$module $func]开头
2. message 大写字母开头
3. 输出参数使用keysAndValues
例如:logging.Warn("[HotSpot onRuleUpdate] Ignoring the frequent param flow rule due to unsupported control behavior", "rule", r)

**Debug:**

1. 日志以[$module $func]开头
2. message 大写字母开头
3. 输出参数使用keysAndValues
例如:logging.Debug("[HotSpot onRuleUpdate] Ignoring the frequent param flow rule due to bad generated traffic controller", "rule", r)

### Special notes for reviews

### CODE REVIEW

File: core/circuitbreaker/circuit_breaker.go [hunk 1]
```diff
@@ -341,21 +341,21 @@ func (s *slowRequestLeapArray) ResetBucketTo(bw *sbase.BucketWrap, startTime uin
...
 	mb := curBucket.Value.Load()
 	if mb == nil {
-		logging.Error(errors.New("Current bucket atomic Value is nil"), "")
+		logging.Error(errors.New("current bucket atomic Value is nil"), "Current bucket atomic Value is nil in slowRequestLeapArray.currentCounter()")
 		return nil
 	}
 	counter, ok := mb.(*slowRequestCounter)
 	if !ok {
-		logging.Error(errors.New("Bucket data type error"), "")
+		logging.Error(errors.New("bucket data type error"), "Bucket data type error in  slowRequestLeapArray.currentCounter()")
```
Comment (liqiangz) [inline core/circuitbreaker/circuit_breaker.go:358]: There are extra spaces between "in" and "slowRequestLeapArray".

File: core/stat/base/sliding_window_metric.go [hunk 2]
```diff
@@ -188,12 +188,12 @@ func (m *SlidingWindowMetric) metricItemFromBuckets(ts uint64, ws []*BucketWrap)
 	for _, w := range ws {
 		mi := w.Value.Load()
 		if mi == nil {
-			logging.Error(errors.New("nil BucketWrap"), "Get nil bucket when generating MetricItem from buckets")
+			logging.Error(errors.New("nil BucketWrap"), "Current bucket Value is nil in SlidingWindowMetric.metricItemFromBuckets()")
```
Comment (liqiangz) [inline core/stat/base/sliding_window_metric.go:191]: value should be lowercase？

File: core/stat/base/sliding_window_metric.go [hunk 3]
```diff
@@ -213,12 +213,12 @@ func (m *SlidingWindowMetric) metricItemFromBuckets(ts uint64, ws []*BucketWrap)
 func (m *SlidingWindowMetric) metricItemFromBucket(w *BucketWrap) *base.MetricItem {
 	mi := w.Value.Load()
 	if mi == nil {
-		logging.Error(errors.New("nil BucketWrap"), "Get nil bucket when generating MetricItem from buckets")
+		logging.Error(errors.New("nil BucketWrap"), "Current bucket Value is nil in SlidingWindowMetric.metricItemFromBucket()")
```
Comment (liqiangz) [inline core/stat/base/sliding_window_metric.go:216]: Here is the same as above

File: ext/datasource/file/refreshable_file.go [hunk 4]
```diff
@@ -83,7 +83,7 @@ func (s *RefreshableFileDataSource) Initialize() error {
 					retryCount := 0
 					for {
 						if retryCount > 5 {
-							logging.Error(errors.New("retry failed"), "Fail to retry watch", "sourceFilePath", s.sourceFilePath)
+							logging.Error(errors.New("Retry failed "), "Fail to retry watch", "sourceFilePath", s.sourceFilePath)
```
Comment (liqiangz) [inline ext/datasource/file/refreshable_file.go:86]: Here should be unified into lower case, and there are extra spaces.

File: core/circuitbreaker/circuit_breaker.go [hunk 5]
```diff
@@ -341,21 +341,21 @@ func (s *slowRequestLeapArray) ResetBucketTo(bw *sbase.BucketWrap, startTime uin
...
 	mb := curBucket.Value.Load()
 	if mb == nil {
-		logging.Error(errors.New("Current bucket atomic Value is nil"), "")
+		logging.Error(errors.New("current bucket atomic Value is nil"), "Current bucket atomic Value is nil in slowRequestLeapArray.currentCounter()")
 		return nil
 	}
 	counter, ok := mb.(*slowRequestCounter)
 	if !ok {
-		logging.Error(errors.New("Bucket data type error"), "")
+		logging.Error(errors.New("bucket data type error"), "Bucket data type error in slowRequestLeapArray.currentCounter()")
```
Comment (louyuting) [inline core/circuitbreaker/circuit_breaker.go:358]: You might need to indicate "type assert failed, expect *slowRequestCounter"

File: core/circuitbreaker/circuit_breaker.go [hunk 6]
```diff
@@ -367,12 +367,12 @@ func (s *slowRequestLeapArray) allCounter() []*slowRequestCounter {
...
 		mb := b.Value.Load()
 		if mb == nil {
-			logging.Error(errors.New("Current bucket atomic Value is nil"), "")
+			logging.Error(errors.New("current bucket atomic Value is nil"), "Current bucket atomic Value is nil in slowRequestLeapArray.allCounter()")
 			continue
 		}
 		counter, ok := mb.(*slowRequestCounter)
 		if !ok {
-			logging.Error(errors.New("Bucket data type error"), "")
+			logging.Error(errors.New("bucket data type error"), "Bucket data type error in slowRequestLeapArray.allCounter()")
```
Comment (louyuting) [inline core/circuitbreaker/circuit_breaker.go:375]: Same with uppon

File: core/circuitbreaker/circuit_breaker.go [hunk 7]
```diff
@@ -518,21 +518,21 @@ func (s *errorCounterLeapArray) ResetBucketTo(bw *sbase.BucketWrap, startTime ui
...
 	mb := curBucket.Value.Load()
 	if mb == nil {
-		logging.Error(errors.New("Current bucket atomic Value is nil"), "")
+		logging.Error(errors.New("current bucket atomic Value is nil"), "Current bucket atomic Value is nil in errorCounterLeapArray.currentCounter()")
 		return nil
 	}
 	counter, ok := mb.(*errorCounter)
 	if !ok {
-		logging.Error(errors.New("Bucket data type error"), "")
+		logging.Error(errors.New("bucket data type error"), "Bucket data type error in errorCounterLeapArray.currentCounter()")
```
Comment (louyuting) [inline core/circuitbreaker/circuit_breaker.go:535]: Same with uppon

**Review by louyuting — CHANGES_REQUESTED** [review]

**Review by louyuting — DISMISSED** [review]
Nice work

**Review by louyuting — APPROVED** [review]

**Pull request merged**